aiounifi/controller.py

```diff
@@ -61,8 +61,18 @@ def __init__(
         self.devices = None
         self.wlans = None
 
+        self.headers = None
```

Comment:
I dunno how to use the cookie jar, so this is a sloppy way of getting around that for now.